src/hooks/use-load-image.js

```diff
@@ -11,7 +11,7 @@ const imagePromise = (url) =>
   });
 
 const useLoadImage = (layout) => {
-  const { url } = layout.style.background;
+  const url = layout.style?.background?.url ?? undefined;
```

Comment:
Don't need to do `?? undefined`, but maybe you are getting a lint error?

Reply:
True, no I don't get lint error, removed it.